src/landing/home-stats/infographics/top-list.jsx

```diff
@@ -84,7 +84,7 @@ function defaultHeader(list) {
 /** @param {{ entry: DashboardBlockListEntry }} _ */
 function BlockListEntry({ entry }) {
   const countStr =
-    parseNumberWithCommas(entry.block_count)?.toLocaleString();
+    parseNumberWithCommas(entry.count)?.toLocaleString();
```

Comment:
@thieflord06 to confirm, can we stop parsing comma formatted string numbers now? are these going to come back as plain number types?

Reply:
That's correct, only plain ints are returned across the board.